wiki/Storyboard/Scripting/Audio/de.md

```diff
@@ -1,36 +1,36 @@
----
-outdated_translation: true
-outdated_since: 3edcc9733800c0e9221a2032eede25009608e79d
----
+# Storyboard-Audio
 
-# Storyboard Audio
-
-**Audio Dateien** (WAV, MP3, und OGG) können abgespielt werden. Es ist wie beim Deklarieren von Objekten, keine Kommandos, sie werden also in Triggern oder Schleifen benutzt. Sie können die Audio Datei auf irgendeine der vier Ebenen legen, wodurch es Ihnen möglich ist, dass zum Beispiel nur im Pass oder Fail Status die Audio Datei abgespielt wird.
+**Audio-Dateien** (`.wav`, `.mp3` und `.ogg`) können zu selbst gewählten Zeitpunkten abgespielt werden. Es ist wie beim Deklarieren von Objekten. Sie sind nicht als Kommandos umgesetzt und werden dadurch nicht in Triggern oder Schleifen benutzt. Du kannst die Audio-Dateien auf irgendeine der vier Ebenen legen (jedoch existiert kein Konzept der "Überschneidung", alle Sounds werden gemischt), wodurch die Audio-Dateien zum Beispiel nur im Pass- oder Fail-Status abgespielt werden.
 
 ## Verwendung
 
-Die Form wie ein Audio deklariert wird:
+Die Form, wie ein Audio-Sample deklariert wird, lautet:
+
+`Sample,<Zeit>,<Ebene>,"<Dateipfad>",<Lautstärke>`
+
+wobei:
 
-`Sample,<Zeit>,<Ebene_Zahl>,"<Dateipfad>",<Lautstärke>`
+- `<Zeit>` der Zeitpunkt ist, ab wann die Audio-Datei abgespielt werden soll.
+- `<Ebene>` ein *numerischer Wert* ist, der die jeweilige Ebene angibt, auf die die Audio-Datei gelegt wird. Die gültigen Werte werden unten aufgelistet.
+- `<Dateipfad>` der Name der Datei ist, die aufgerufen werden soll (Datei im Format `.wav`, `.mp3` oder `.ogg`).
+- `<Lautstärke>` eine Zahl von 1 bis 100 ist, welche die jeweilige Lautstärke des Audios angibt (wenn die Zahl weggelassen wird, wird der Wert auf den Standardwert 100 gesetzt).
 
-wo:
+Die zulässigen Werte der Ebenen sind:
 
-- `<Zeit>` ist der Zeitpunkt, ab wann die Audio Datei abgespielt werden soll.
-- `<Ebene_Zahl>` ist ein numerischer Wert, der die jeweilige Ebene angibt, auf die die Audio Datei gelegt wird:
-  - 0 - Background
-  - 1 - Fail
-  - 2 - Pass
-  - 3 - Foreground
-- `<Dateipfad>` ist der Name der Datei, den Sie aufrufen wollen.
-- `<Lautstärke>` ist eine Zahl von 1 bis 100, welches die jeweilige Lautstärke des Audios angibt (wenn die Zahl weggelassen wird, wird der Wert auf 100 gesetzt).
+| Wert | Ebene |
+| :-: | :-- |
+| 0 | Hintergrund |
+| 1 | Fail |
+| 2 | Pass |
+| 3 | Vordergrund |
 
 ## Beispiel
 
-In diesen Beispiel soll am Ende einer Pass Sektion eine Musikdatei abgespielt werden und spielt dann wenige Sekunden später einen Soundeffekt ab:
+In diesen Beispiel soll beim Bestehen der Beatmap (z. B. beim Erreichen des Pass-Status) eine Musikdatei mit dem Geräuscheffekt einer Explosion als Teil der Szene abgespielt werden:
```

Comment:
```suggestion
In diesen Beispiel soll beim Bestehen der Beatmap (z. B. beim Erreichen des Pass-Status) ein (kurzes!) Musikstück und anschließend ein Soundeffekt einer Explosion als Teil der Szene abgespielt werden:
```
Reads a bit better in my opinion.

Reply:
I'm further editing your idea, as I spotted a typo:

```suggestion
In diesem Beispiel soll beim Bestehen der Beatmap (z. B. beim Erreichen des Pass-Status) eine Musikdatei mit dem Geräuscheffekt einer Explosion als Teil der Szene abgespielt werden:
```